install.sh

```diff
@@ -3683,19 +3665,15 @@ mark_directory_as_target() {
         return
     fi
 
-    if [ "$2" != "" ]; then
```

Comment:
Thats a but, here we add the `$2` below we  check for writable -> not according to the doc, its strange behavior.
Here **only* the first dir is checked, Thats so far what I could made up by looking at the calls, exactly what should have been achieved.... These caveats came in related to the coreHooksPath code paths i think....